ext/oci8/tests/extauth_01.phpt

```diff
@@ -70,7 +66,7 @@ var_dump($c);
 
 echo "Test 7\n";
 
-$c = oci_connect('/', '', 'anything', null, OCI_CRED_EXT);
+$c = oci_connect('/', '', 'localhost', null, OCI_CRED_EXT);
```

Comment:
when `localhost` is used instead of `anything`, tests fail, so it cannot be really "anything".